test/integration/narrow/routes/business-details.test.js

```diff
@@ -0,0 +1,90 @@
+import { describe, test, expect } from 'vitest'
```

Comment:
Introduced Cheerio package to test what the UI is rendering in an integration test